sql/testdata/builtin_function

```diff
@@ -332,3 +332,56 @@ SELECT extract(nanosecond from timestamp '2001-04-10 12:04:59.34565423')
 
 query error extract: unsupported timespan: nansecond
 SELECT extract(nansecond from timestamp '2001-04-10 12:04:59.34565423')
+
+query T
+SELECT abs(-1.0)
+----
+1
+
+query T
+SELECT abs(-0.0)
+----
+0
+
+query T
+SELECT abs(1.0)
+----
+1
+
+query error abs: abs of MinInt64 is not defined
+SELECT abs(-9223372036854775808)
+
+query T
+SELECT abs(-1)
+----
+1
+
+query T
+SELECT abs(0)
+----
+0
+
+query T
+SELECT abs(1)
+----
+1
+
+query T
+SELECT pi()
+----
+3.141592653589793
+
+query T
+SELECT sin(0.0)
+----
+0
+
+query T
+SELECT sin(1.0)
+----
+0.8414709848078965
+
+query T
+SELECT sin(1::float)
```

Comment:
This is going to be equivalent to the previous test as long as casting works (which is tested elsewhere). Perhaps replace with a test for `sin(pi())`.